index.js

```diff
@@ -0,0 +1,6270 @@
+     
+const canvasWith = 512;
+let canvas = document.getElementById('draw-canvas');
+ function drawCanvasImage () {   
+    
+    let ctx = canvas.getContext('2d');
+    ctx.clearRect(0, 0, 512, 512);
+    let img = new Image();
+    img.src = "image.png";
+    img.onload = function() {    
+        ctx.drawImage(img,0,0,canvasWith,canvasWith);
+      }  
+    
+} 
+
+ function drawCanvasRgba () {
+   
+   let ctx = canvas.getContext('2d');
+    ctx.clearRect(0, 0, canvasWith, canvasWith);
+    let x = 0, y = 0; color ='', elementWith = 0, element = [];
```

Comment:
if variables are not reassigned use "const" instead of "let". For each variable separate declaration is needed.